Project.toml

```diff
@@ -1,28 +1,21 @@
 name = "Metalhead"
 uuid = "dbeba491-748d-5e0e-a39e-b530a07fa0cc"
-version = "0.5.1"
+version = "0.5.3"
 
 [deps]
 BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
-ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
 Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
-ImageFiltering = "6a3955dd-da59-5b1f-98d4-e7296123deb5"
-Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
-REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
-Requires = "ae029012-a4dd-5104-9daa-d747884805df"
-Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 
 [compat]
-BSON = "0.2"
-ColorTypes = "0.8, 0.9"
-Flux = "0.10, 0.11"
-ImageFiltering = "0.6"
-Images = "0"
-Requires = "0.5, 1.0"
+BSON = "0.3.2"
 julia = "1.4"
```

Comment:
should add Flux v0.12 as lower bound?

Reply:
Yes will do that once the release is tagged (had some problems withe the `-DEV`...I was probably doing something wrong).